ext/GeometryOpsProjExt/reproject.jl

```diff
@@ -55,6 +59,6 @@ function reproject(geom, transform::Proj.Transformation; time=Inf, threaded = Fa
         transforms = ThreadFunctors(functors, tasks_per_thread)
         return apply(transforms, GI.PointTrait(), geom; crs=target_crs, kw...)
     else # threaded isa False
-        return apply(ApplyToPoint{_is3d(geom)}(transform), GI.PointTrait(), geom; time, target_crs, kw...)
+        return apply(ApplyToPoint{_is3d(geom)}(transform), GI.PointTrait(), geom; crs = target_crs, kw...)
```

Comment:
I think it may be better to still make an if else block for these, putting _is3d in the type directly may still mean instability even though there are only two options

Reply:
can do then